dll/win32/rasdlg/lang/ro-RO.rc

```diff
@@ -106,50 +106,50 @@ BEGIN
 CAPTION "Schimbare parolă"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Parola de conectare la rețea a expirat.\nIntroduceți o parolă nouă, apoi apăsați Confirmă pentru a actualiza informațiile de autentificare.", 1062, 7, 5, 210, 24
+    LTEXT "Parola de conectare la rețea a expirat.\nIntroduceți o parolă nouă, apoi apăsați OK pentru a actualiza informațiile de autentificare.", 1062, 7, 5, 210, 24
     LTEXT "&Parola nouă:", 1064, 8, 43, 108, 8
     EDITTEXT 1060, 118, 38, 98, 14, ES_PASSWORD | ES_AUTOHSCROLL
     LTEXT "&Confirmați parola:", 1061, 8, 61, 102, 8
     EDITTEXT 1058, 118, 57, 98, 14, ES_PASSWORD | ES_AUTOHSCROLL
-    DEFPUSHBUTTON "Con&firmă", 1, 90, 84, 60, 14, WS_GROUP
-    PUSHBUTTON "A&nulează", 2, 154, 84, 60, 14
+    DEFPUSHBUTTON "&OK", 1, 90, 84, 60, 14, WS_GROUP
+    PUSHBUTTON "&Revocare", 2, 154, 84, 60, 14
 END
 
 109 DIALOGEX 6, 18, 223, 122
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUPWINDOW | WS_VISIBLE | WS_CAPTION
 CAPTION "Schimbare parolă"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Parola de conectare la rețea a expirat.\nIntroduceți o parolă nouă, apoi apăsați Confirmă pentru a actualiza informațiile de autentificare.", 1062, 7, 5, 212, 29
+    LTEXT "Parola de conectare la rețea a expirat.\nIntroduceți o parolă nouă, apoi apăsați OK pentru a actualiza informațiile de autentificare.", 1062, 7, 5, 212, 29
     LTEXT "&Vechea parolă:", 1063, 7, 42, 108, 8
     EDITTEXT 1059, 116, 38, 100, 14, ES_PASSWORD | ES_AUTOHSCROLL
     LTEXT "&Noua parolă:", 1064, 7, 60, 108, 8
     EDITTEXT 1060, 116, 56, 100, 14, ES_PASSWORD | ES_AUTOHSCROLL
     LTEXT "&Confirmați parola:", 1061, 7, 78, 108, 8
     EDITTEXT 1058, 116, 74, 100, 14, ES_PASSWORD | ES_AUTOHSCROLL
-    DEFPUSHBUTTON "Con&firmă", 1, 92, 101, 60, 14, WS_GROUP
-    PUSHBUTTON "A&nulează", 2, 156, 101, 60, 14
+    DEFPUSHBUTTON "&OK", 1, 92, 101, 60, 14, WS_GROUP
+    PUSHBUTTON "&Revocare", 2, 156, 101, 60, 14
 END
 
 110 DIALOGEX 12, 16, 230, 105
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUPWINDOW | WS_VISIBLE | WS_CAPTION
 CAPTION "Retroapelare"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Aveți aleasă opțiunea numărului specificat de către apelat. Apăsați Confirmă pentru a fi apelat de către server la numărul de mai jos. Apăsați Anulează pentru a omite retroapelarea.", 1066, 5, 5, 221, 39, SS_NOPREFIX
+    LTEXT "Aveți aleasă opțiunea numărului specificat de către apelat. Apăsați OK pentru a fi apelat de către server la numărul de mai jos. Apăsați Revocare pentru a omite retroapelarea.", 1066, 5, 5, 221, 39, SS_NOPREFIX
     LTEXT "Int&roduceți numărul de telefon al modemului dumneavoastră:", 1067, 5, 46, 222, 8, NOT WS_GROUP
     EDITTEXT 1065, 5, 57, 221, 14, ES_AUTOHSCROLL
-    DEFPUSHBUTTON "Con&firmă", 1, 100, 85, 60, 14, WS_GROUP
-    PUSHBUTTON "A&nulează", 2, 164, 85, 60, 14
+    DEFPUSHBUTTON "&OK", 1, 100, 85, 60, 14, WS_GROUP
+    PUSHBUTTON "&Revocare", 2, 164, 85, 60, 14
```

Comment:
This change introduces an accelerator collision with the line
 LTEXT "Int&roduceți numărul de telefon al modemului dumneavoastră:", 1067, 5, 46, 222, 8, NOT WS_GROUP
Please check your romanian Windows to find out which of both lines need to be changed.